docs: update Getting Started for React Native 0.60 (#687)
kmagiera/react-native-gesture-handler: docs/getting-started.md

- Mechanisms to use touchables that run in native thread and follow platform default behavior; e.g. in the event they are in a scrollable component, turning into pressed state is slightly delayed to prevent it from highlighting when you fling.
- The possibility to implement smooth gesture interactions thanks to Animated Native Driver &mdash; interactions will be responsive even when the JS thread is overloaded.

## Installation

Since the library uses native support for handling gestures, it requires an extended installation to the norm.

### With [React Native](http://facebook.github.io/react-native/) app (no Expo)

#### Requirements

| version | `react-native` version |
| ------- | ---------------------- |
| 1.1.0+  | 0.57.2+                |
| <1.1.0  | 0.50.0+                |

Note that if you wish to use [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html) support for [interactions](interactions.md) you need to use v16.3 of [React](https://reactjs.org/)

#### JS

First, install the library using `yarn`:

```bash
yarn add react-native-gesture-handler
```

or with `npm` if you prefer:

```bash
npm install --save react-native-gesture-handler
```

#### Linking

> > **Important**: This step is only applicable for React Native 0.59 or lower. Since v0.60 linking happens automatically.

Reviewer: I'd rephrase it to

```suggestion
> **Important**: You only need to do this step if you're using React Native 0.59 or lower. Since v0.60, linking happens automatically.
```

}
```

##### Running jetifier

> > **Important:** This step is only necessary if you're on React Native 0.60 or greater.

Reviewer: I'd rephrase it to

```suggestion
> **Important:** You only need to do this step if you're on React Native 0.60 or greater.
```

React Native 0.60 migrated from Support Library to AndroidX. React Native Gesture Handler is not yet compatible with AndroidX. Until then, please use [`jetifier`](https://www.npmjs.com/package/jetifier) to make it work with newest React Native:

```sh

> yarn add jetifier

Reviewer: ```suggestion
yarn add --dev jetifier
```

```

or with npm:

```sh
npm install --save-dev jetifier
```

> We recommend using "postinstall" phase to run it. You can add it in your `package.json` file:

Reviewer: I prefer to add this step **before** running `yarn add jetifier` so it's automatically run. Or we need to include extra step `yarn jetify` after this step so the dep is jetified.

Author: Oh yea, good call
